tests/python/common/test_homophily.py

```diff
@@ -0,0 +1,19 @@
+import unittest
+
+import backend as F
+
+import dgl
+from test_utils import parametrize_idtype
+
+
+@unittest.skipIf(dgl.backend.backend_name == "tensorflow", reason="Skip TF")
+@parametrize_idtype
+def test_node_homophily(idtype):
+    # NOTE: If you want to update this test case, remember to update the
```

Comment:
If we want to introduce this practice, let's follow this format:

```
# IfChangeThenChange: python/dgl/homophily.py
# Update the docstring example.
```

Reply:
Added in the conventions:
https://docs.google.com/document/d/1NGXPg4U67ZpMDVxxkb-SprEx11d8zqNB_0cLweTL90Y/edit#

Reply:
updated